src/index.html

```diff
@@ -1,13 +1,709 @@
+            >
+              <div class="explore__box">
+                <div class="explore__photo-container">
+                  <img
+                    src="./images/photos/photo10.png"
+                    alt="Travel"
+                    class="explore__box-photo"
+                    data-aos="flip-right"
+                    data-aos-duration="2000"
+                  >
+                </div>
+
+                <h3 class="explore__box-sub-title">
+                  <a href="#" class="explore__link">
+                    Travel
+                  </a>
+                </h3>
+
+                <p class="explore__box-text">
+                  Travel with The Met connects
+                  the Museum's wide and diverse
+                  audience to art, inspiration,
+                  and historical perspective.
+                </p>
+              </div>
+            </div>
+
+            <div
+              class="explore__item
+              grid__item
+              grid__item--tablet-2-5
+              grid__item--desktop-9-12"
+            >
+              <div class="explore__box">
+                <div class="explore__photo-container">
+                  <img
+                    src="./images/photos/photo9.png"
+                    alt="Virtual group tours"
+                    class="explore__box-photo"
+                    data-aos="flip-right"
+                    data-aos-duration="2000"
+                  >
+                </div>
+
+                <h3 class="explore__box-sub-title">
+                  <a href="#" class="explore__link">
+                    Virtual group tours
+                  </a>
+                </h3>
+
+                <p class="explore__box-text">
+                  Request a guided tour, reserve admission
+                  and lead your own small group, or request
+                  a live virtual tour led by a Museum guide.
+                </p>
+
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contacts">
+        <div class="container">
+          <h2
+            class="page__section-title
+            contact-us__title"
+          >
+            Contact us
+          </h2>
+
+          <div
+            class="grid
+            grid--desktop
+            grid--tablet
+            grid__gap"
+          >
+            <div
+              class="grid__item
+              grid__item--tablet-1-2
+              grid__item--desktop-1-3"
+            >
+              <h4 class="contact-us__address">
+                The Met Fifth Avenue
+              </h4>
+
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+
+                <a
+                  href="tel:+1 212-535-7710"
+                  class="contact-us__info-link"
+                >
+                  +1 212-535-7710
```

Comment:
Links of contacts section change their color on hover immediately. According to the checklist all transitions should have the same time

![image](https://user-images.githubusercontent.com/78492113/218104433-3e282b02-c59b-46c8-8ea3-9503959ced5b.png)